integration/pom.xml

```diff
@@ -97,6 +97,13 @@
         </dependency>
 
         <dependency>
+            <groupId>com.mulesoft.mule.modules</groupId>
```

Comment:
I told you that a CE component cannot use a EE component. You can't use this dependency here